web/migrations/0053_selfcheckin_sessionfeedback_sessionsurvey.py

```
@@ -0,0 +1,167 @@
+                        validators=[
+                            django.core.validators.MinValueValidator(1),
+                            django.core.validators.MaxValueValidator(5),
+                        ],
+                    ),
+                ),
+                ("notes", models.TextField(blank=True)),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                (
+                    "session",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, related_name="checkins", to="web.session"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)